gradle-plugin/src/main/kotlin/com/improve_future/harmonica/task/JarmonicaTaskMain.kt

```
@@ -47,4 +55,54 @@ abstract class JarmonicaTaskMain {
+                }
+                .mapNotNull {
+                    loadClass(it.name.removeSuffix(".class").replace('/', '.'))
+                }
+                .toList()
+        }
+    }
+
+    private fun loadClass(className: String): Class<*>? {
+        return try {
+            Class.forName(className, false, classLoader)
+        } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)